### PR TITLE
Update python.md

### PR DESCRIPTION
**Pull request by zzjjbb**
- fix errors about "typing" in Python
- fix a formatting error

**审核的同学** 请着重关注以下四方面：

1. 注意有没有 typo
2. 不论你是否熟悉相关知识，都请以初学者的角度把这个 PR 的内容阅读一遍，跟着作者的思路走，然后谈谈你的感受
3. 如果你熟悉相关知识，请按照自己的理解评估这个 PR 的内容是否合适
4. 请**尽量**保持跟进直到它被 merge 或 close

### CODE REVIEW

**Comment by ksyx** [comment]
请阅读 [如何参与](https://oi-wiki.org/intro/htc/) 及 [格式手册](https://oi-wiki.org/intro/format/) 以方便 PR 更顺利地被合并，thx

File: docs/lang/python.md [hunk 1]
```diff
@@ -348,9 +348,9 @@ def circumference(radius: float) -> float:
     return 2 * math.pi * radius
 ```
 
-当然，除了函数函数，变量也是可以类型标注的，你可以通过调用 `__annotations__` 来查看函数中所有的类型标注。
+自 Python 3.6 起 ([PEP 526](https://www.python.org/dev/peps/pep-0526/))，除了函数函数，变量也是可以类型标注的，你可以通过调用 `__annotations__` 来查看模块和类中所有的变量类型标注。
 
-```python3
+```python3.6+
```
Comment (CoelacanthusHex) [inline docs/lang/python.md:353]: There is no language named `python3.6`, you should still use `python3`

File: docs/lang/python.md [hunk 2]
```diff
@@ -582,126 +582,128 @@ if __name__ == '__main__':
 
 完整的代码如下：
 
-??? note "C++"
```
Comment (CoelacanthusHex) [inline docs/lang/python.md:585]: Why remove note? Please tell me the reason!

**Review by CoelacanthusHex — CHANGES_REQUESTED** [review]

File: docs/lang/python.md [hunk 3]
```diff
@@ -339,7 +339,7 @@ Success: no issues found in 1 source file
 
 ### 类型标注
 
-类型标注是自 Python 3.0 引入的特征，是添加类型提示的重要方法。例如这段代码就引入了类型标注，你可以通过调用 `circumference.__annotations__` 来查看函数中所有的类型标注。
+类型标注是自 Python 3.5 引入的特征 ([PEP 484](https://www.python.org/dev/peps/pep-0484/))，是添加类型提示的重要方法。例如这段代码就引入了类型标注，你可以通过调用 `circumference.__annotations__` 来查看函数中所有的类型标注。
```
Comment (CoelacanthusHex) [inline docs/lang/python.md:342]: Strictly speaking, this feature was first introduced in Python 3.0 by [PEP 3107](https://www.python.org/dev/peps/pep-3107/), after [PEP 484](https://www.python.org/dev/peps/pep-0484/), it is basically available in [PEP 526](https://www.python.org/dev/peps/pep-0526/).

**Review by CoelacanthusHex — CHANGES_REQUESTED** [review]

File: docs/lang/python.md [hunk 4]
```diff
@@ -348,9 +348,9 @@ def circumference(radius: float) -> float:
     return 2 * math.pi * radius
 ```
 
-当然，除了函数函数，变量也是可以类型标注的，你可以通过调用 `__annotations__` 来查看函数中所有的类型标注。
+自 Python 3.6 起 ([PEP 526](https://www.python.org/dev/peps/pep-0526/))，除了函数函数，变量也是可以类型标注的，你可以通过调用 `__annotations__` 来查看模块和类中所有的变量类型标注。
```
Comment (Xeonacid) [inline docs/lang/python.md:351]:
```suggestion
自 Python 3.6 起 ([PEP 526](https://www.python.org/dev/peps/pep-0526/))，除了函数参数，变量也是可以类型标注的，你可以通过调用 `__annotations__` 来查看模块和类中所有的变量类型标注。
```

**Comment by zzjjbb** [comment]
Thanks for reviewing this! I think most part of this PR contains problems and is kind of confusing. I'll submit a new one instead.

**Pull request closed without merging**